### PR TITLE
call updater from notifier

### PR DESCRIPTION
**Pull request by d-loose**
Just to show that this works now :)

(assumes that the firmware-updater binary can be found in PATH)

https://user-images.githubusercontent.com/113362648/197782811-185df44a-2b16-489f-92f5-5bae4cd6060b.mp4

### CODE REVIEW

File: packages/firmware_notifier/bin/firmware_notifier.dart [hunk 1]
```diff
@@ -7,7 +9,8 @@ void main(List<String> arguments) async {
   final notifications = await sendUpdateNotifications(client, updates);
 
   notifications.map((n) => n.action).forEach((action) async {
-    print('<run command for deviceId, version ${await action}>');
+    final args = (await action).split(', ');
+    await Process.run('firmware-updater', args);
```
[the remark this answers is not in the document]
Comment (d-loose) [reply packages/firmware_notifier/bin/firmware_notifier.dart:13]: Wouldn't this give me the path of the notifier's binary (which is separate from the firmware updater)?

[the remark this answers is not in the document]
Comment (jpnurmi) [reply packages/firmware_notifier/bin/firmware_notifier.dart:13]: yes! \*brainfart\*

**Comment by d-loose** [comment]
snapcraft.yaml updates:

- add dbus slot for `com.canonical.firmware_updater` so that the gtk_application widgets are allowed to listen on dbus
- install `dart` after building `flutter-git`
- build `firmware-notifier` as separate app within the snap
- make `firmware-updater` a user daemon that can be dbus-activated
Problem: snapcraft doesn't create a `/snap/bin/firmware-updater` command even though it's using this command in the `.desktop` file it generates. (can be launched with `gapplication launch com.canonical.firmware_updater` instead)

**Comment by d-loose** [comment]
Until we sort out DBus activation issues with Snap/Flutter/GApplication, let's simply call `firmware-updater <deviceID> <version>` directly from the notifier.

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

**Pull request merged**